config/locales/views/landing_page/de.yml

```diff
@@ -23,25 +23,25 @@ de:
   items:
     messages:
       successfully_created: "Item wurde erfolgreich erstellt."
-      successfully_updated: "Item wurde erfolgreich geupdatet."
+      successfully_updated: "Item wurde erfolgreich aktualisiert."
       successfully_destroyed: "Item wurde erfolgreich zerstört."
       successfully_borrowed: "Item wurde erfolgreich ausgeliehen"
       successfully_returned: "Item wurde erfolgreich zurückgegeben"
       successfully_returned-by-owner: "Item wurde erfolgreich vom Besitzer zurückgegeben"
       successfully_reserved: "Item wurde erfolgreich reserviert"
       unsuccessfully_reserved: "Item Reservierung fehlgeschlagen"
-      joining_waitlist_succeeded: "Du bist der Warteliste beigetreten."
-      leaving_waitlist_succeeded: "Du hast die Warteliste verlassen."
-      lending_error: "Ausleihsystem Fehler"
-      not_allowed_to_destroy: "Dir ist nicht erlaubt das Item zu löschen."
-      not_allowed_to_download: "Dir ist nicht erlaubt den QR-Code des Items herunterzuladen."
-      not_allowed_to_edit: "Dir ist nicht erlaubt das Item zu bearbeiten."
-      not_allowed_to_access: "Dir ist nicht erlaubt auf das Item zu zugreifen."
+      joining_waitlist_succeeded: "Sie sind der Warteliste beigetreten."
+      leaving_waitlist_succeeded: "Sie haben die Warteliste verlassen."
+      lending_error: "Ausleihsystem-Fehler"
+      not_allowed_to_destroy: "Ihnen ist nicht erlaubt, das Item zu löschen."
+      not_allowed_to_download: "Ihnen ist nicht erlaubt, den QR-Code des Items herunterzuladen."
+      not_allowed_to_edit: "Ihnen ist nicht erlaubt, das Item zu bearbeiten."
+      not_allowed_to_access: "Ihnen ist nicht erlaubt, auf das Item zuzugreifen."
       deletion_confirmation: "Soll das Item wirklich gelöscht werden?"
-      range_error: "muss eine gültige Zahl oder ein Zahlenbereich sein, z. B. 69-420"
+      range_error: "muss eine gültige Zahl oder ein Zahlenbereich sein, z.B. 69-420"
```

Comment:
Wenn wir DIN-konform sein wollen, sollte das Leerzeichen bleiben. ( https://de.wikipedia.org/wiki/DIN_5008#Abk%C3%BCrzungen_(Abschnitt_4.5) )

Reply:
PO wants to conform to DIN5008

Resolved by 2b4f37d